API/common/utils.py

```diff
@@ -253,63 +253,54 @@ def remove_file(filename):
         pass
 
 
-def get_section_sizes_from_map(mapfile):
+_liblist = [
+    'libhttpparser.a',
+    'libiotjs.a',
+    'libjerry-core.a',
+    'libjerry-ext.a',
+    'libjerry-port-default.a',
+    'libjerry-port-default-minimal.a',
+    'libtuv.a'
+]
+
+
+def calculate_section_sizes(builddir):
     '''
-    Returns the sizes of the main sections.
+    Return the sizes of the main sections.
     '''
-    sizes = {
+    section_sizes = {
         'bss': 0,
         'text': 0,
         'data': 0,
-        'rodata': 0,
-        'total': 0
```

Comment:
I've removed the `total` from the size information, because the `sizes['total'] = sizes['text'] + sizes['data'] + sizes['rodata']` logic will be moved to webpage side.